### PR TITLE
Implement event source monitoring with app-specific heuristics 

### LINKED ISSUE #205: Support new app: Notion and Apple Notes
### BundleId of the apps

* `notion.id`
* `com.apple.Notes`

### PR DESCRIPTION
**Pull request by starbugs**
Fixes #205.

### CODE REVIEW

File: WakaTime/Helpers/MonitoringManager.swift [hunk 1]
```diff
@@ -89,9 +92,12 @@ class MonitoringManager {
                     entity: title,
                     category: .planning)
             case .notes:
-                return HeartbeatData(
-                    entity: title,
-                    category: .learning)
+                if element.rawTitle == "Notes" {
+                    return HeartbeatData(
+                        entity: title,
+                        category: .learning
```
[the remark this answers is not in the document]
Comment (alanhamlett) [reply WakaTime/Helpers/MonitoringManager.swift:98]: https://wakatime.com/colors/categories

[the remark this answers is not in the document]
Comment (starbugs) [reply WakaTime/Helpers/MonitoringManager.swift]: All fixed in ee1f9cd3d5e525c5fb6362eedbeb666851572f55

[the remark this answers is not in the document]
Comment (starbugs) [reply WakaTime/Helpers/MonitoringManager.swift]: I left Slack in the Electron list as well

**Review by alanhamlett — APPROVED** [review]
🚀

**Pull request merged**